test/minikube/minikube_base_database_test.go

```diff
@@ -221,7 +221,7 @@ func TestKubernetesBasicDatabase(t *testing.T) {
 		})
 
 		t.Run("verifySecret", func(t *testing.T) { verifySecret(t, namespaceName) })
-		t.Run("verifyDBHeadlessService", func(t *testing.T) { verifyDBService(t, namespaceName, admin0, headlessServiceName), true })
+		t.Run("verifyDBHeadlessService", func(t *testing.T) { verifyDBService(t, namespaceName, admin0, headlessServiceName, true) })
```

Comment:
did this somehow get committed by mistake?

Reply:
Ah I have not merged master so it picked it up as a diff... fixing